_layouts/default.html

```diff
@@ -29,15 +29,15 @@
         <meta property="twitter:site" content="{{ langdata.enabled }}">
         <meta name="twitter:creator" content="@bisq_network">
         <link rel="canonical" href="{{ page.url }}">
```

Comment:
canonical href should be the localized URL, for example /ja/dao/ when viewing /ja/dao/ - currently it's /dao/